client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -248,12 +248,15 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
 
   private val messagesHelper: TransportMessagesHelper = new TransportMessagesHelper()
 
+  private val extraInfo = ApplicationInfoProvider.instantiate(conf)
+    .provide() + (CelebornConf.QUOTA_INTERRUPT_SHUFFLE_ENABLED.key -> conf.quotaInterruptShuffleEnabled.toString)
```

Comment:
Should move `(CelebornConf.QUOTA_INTERRUPT_SHUFFLE_ENABLED.key -> conf.quotaInterruptShuffleEnabled.toString)` to `DefaultApplicationInfoProvider#provide()` ?